pages/index.tsx

```diff
@@ -105,27 +105,28 @@ const Home: NextPage = () => {
     return (
       <div key={`tree${innerNode.id}`}>
         <p style={{ marginLeft: `${level * 30}px` }}>
-          <span>{innerNode.expanded ? '⬇️' : '➡️'}</span>
-          <i>• </i>
+          <a
+            onClick={() => toggleExpanded(innerNode)}
+            className="is-unselectable"
+          >
+            {innerNode.expanded ? '⮟' : '➤'}
+          </a>
```

Comment:
If we want to be sematically correct we should not use a tags here as it is reserved for hyper links. The correct element to use here would be a button.

Reply:
:+1: seems reasonable,will keep the a tag since it formats the arrow differently compared to having it as a normal "text" : )